src/main/java/edu/uc/inventorymanager/dto/Item.java

```diff
@@ -17,15 +18,19 @@
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     @Column(nullable = false)
+    @SerializedName("id")
     private int id;
 
     @Column(nullable = false)
     @NonNull
+    @SerializedName("name")
     private String name;
 
     @NonNull
+    @SerializedName("description")
     private String description;
 
+    @SerializedName("location")
     private String location;
```

Comment:
Added @searializedName(“ ”)
Since this DTO class is being used to connect to a database, I believe it’s a good idea to add @SerializedName annotation. This will make serializing and deserializing much easier to perform. 
Helpful link - https://www.javadoc.io/doc/com.google.code.gson/gson/2.6.2/com/google/gson/annotations/SerializedName.html